LEDStateArray leds
---
bool success
string message